configure-x86.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#---------------------------------------------------------------------
+# Description: This script is the *ONLY* place where "qemu*" build options
+# should be defined.
+#
+# Note to maintainers:
+#
+# XXX: Every option group *MUST* be documented explaining why it has
+# been specified.
+#---------------------------------------------------------------------
+
+typeset -a qemu_options
+
+qemu_version_file=VERSION
+if [ ! -f ${qemu_version_file} ];then
+	echo "QEMU version file not found"
+	exit 1
+fi
+
+qemu_version_major=$(cat ${qemu_version_file} | cut -d. -f1)
+qemu_version_minor=$(cat ${qemu_version_file} | cut -d. -f2)
+
+#---------------------------------------------------------------------
+# Disabled options
+
+# bluetooth support not required
+qemu_options+=(--disable-bluez)
+
+# braille support not required
+qemu_options+=(--disable-brlapi)
+
+# Don't build documentation
+qemu_options+=(--disable-docs)
+
+# Disable GUI (graphics)
+qemu_options+=(--disable-curses)
+qemu_options+=(--disable-gtk)
+qemu_options+=(--disable-opengl)
+qemu_options+=(--disable-sdl)
+qemu_options+=(--disable-spice)
+qemu_options+=(--disable-vte)
+
+# Disable graphical network access
+qemu_options+=(--disable-vnc)
+qemu_options+=(--disable-vnc-jpeg)
+qemu_options+=(--disable-vnc-png)
+qemu_options+=(--disable-vnc-sasl)
+
+# Disable unused filesystem support
+qemu_options+=(--disable-fdt)
+qemu_options+=(--disable-glusterfs)
+qemu_options+=(--disable-libiscsi)
+qemu_options+=(--disable-libnfs)
+qemu_options+=(--disable-libssh2)
+
+# Disable unused compression support
+qemu_options+=(--disable-bzip2)
+qemu_options+=(--disable-lzo)
+qemu_options+=(--disable-snappy)
+
+# SECURITY: Disable unused security options
+qemu_options+=(--disable-seccomp)
+qemu_options+=(--disable-tpm)
+
+# Disable userspace network access ("-net user")
+qemu_options+=(--disable-slirp)
+
+# Disable USB
+qemu_options+=(--disable-libusb)
+qemu_options+=(--disable-usb-redir)
+
+# SECURITY: Don't build a static binary (lowers security)
+# needed if qemu >2.7
```

Comment:
needed if `qemu > 2.7` but it actually sets `--disable-static` when qemu < 2.7? I see that `--disable-static` is already default in v2.9. So it is indeed needed when `qemu < 2.7` right? Please update the comment accordingly.

Reply:
oops, sorry fixing comment, thanks